### PR TITLE
Use redux in React components

### PR DESCRIPTION
**Pull request by estebanmual**
In this milestone I: 
  - Make sure that state is immutable in reducers!
  - Import Redux Provider and the store into the main component.
  - Connect the app to the Redux store.
  - Make sure that list of books displays books from the Redux store.
  - Use useSelector to consume the state.
  - In the React component responsible for adding new books, add functionality for the "Add new" button click.
  - In the React component responsible for removing books, add functionality for the "Remove" button click.
  - In the React component responsible for categories, add functionality for the "Check status" button click.

### CODE REVIEW

File: src/redux/categories/categories.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { STATUS_CATEGORIES } from '../../constants';
 export default function categoriesReducer(state = {}, action) {
   switch (action.type) {
     case STATUS_CATEGORIES:
+      console.log('Under construction');
```
Comment (Meltrust) [inline src/redux/categories/categories.js]: - [x] Please, remove this debugging piece of code `console.log` in order to improve the cleanliness of the repo.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @estebanmual,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice use of action creators and local state✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/categories/categories.js (hunk 1 above)
Comment (estebanmual) [reply src/redux/categories/categories.js]: @Meltrust Thank you so much for that feedback, I forgot to delete it. I appreciate it!  😄

**Review by Meltrust — APPROVED** [review]
Hi @estebanmual,


## Wow, you did it 🎉
Brilliant
## Thank you for deleting the debugging code, as requested in the previous review 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**